src/cli.js

```diff
@@ -315,6 +316,7 @@ async function runCmd (argv, stdout, stderr) {
           startTime = Date.now();
           stdout.write('File change, rebuilding...\n');
         });
+        return true;
```

Comment:
Is this correct? You return a promise below but here you return a boolean?

Reply:
Yes, this is a result value to indicate we're going into watch mode.